code/backend/batch/requirements.txt

```diff
@@ -1,18 +1,28 @@
-azure-identity==1.15.0
-Flask==3.0.2
-openai==0.27.8
-azure-storage-blob==12.19.0
+azure-functions==1.18.0
+streamlit==1.30.0
+openai==1.6.1
+matplotlib==3.8.2
+plotly==5.18.0
+scipy==1.11.4
+scikit-learn==1.3.2
+transformers==4.36.2
 python-dotenv==1.0.1
-langchain==0.1.4
 azure-ai-formrecognizer==3.3.2
+azure-storage-blob==12.19.0
+azure-identity==1.15.0
+azure-ai-contentsafety==1.0.0
 requests==2.31.0
 tiktoken==0.5.2
 azure-storage-queue==12.9.0
+langchain==0.1.4
 beautifulsoup4==4.12.3
 fake-useragent==1.4.0
 chardet==5.2.0
 --extra-index-url https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-python/pypi/simple/
 azure-search-documents==11.4.0b8
 opencensus-ext-azure==1.1.13
-azure-ai-contentsafety==1.0.0
+pandas==1.5.1
 python-docx==1.1.0
+azure-keyvault-secrets==4.4.*
+# Add dev dependencies here - this will be refactored out by Poetry
```

Comment:
pytest has been moved to dev-requirements.txt - can you check all the versions as dependabot has been turned on and some of these versions have been updated. e.g. scipy, scikit-learn, transformers, etc

Reply:
Updated.